Eastern-Research-Group/expert-query
# Feature/113 static input fields

## Related Issues:
* [EQ-113](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-113)

## Main Changes:
* Added inputs for all relevant domain values.
* Added client-side aliases to the `domainValues` context object for fields that share domain values with other fields.
* Restructured the state management to reduce boilerplate.
* Added a mapping file for the filter fields to facilitate their rendering.
* Removed `locationText` from the domain values. This domain has a large number of values, and filtering them will require querying the database.
* Added logic for removing duplicate values from the set retrieved from the domain service.

## Steps To Test:
1. Run the domain values ETL task.
2. Navigate to the app's home page: [http://localhost:3000](http://localhost:3000)
3. Confirm that the `assessments` and `actions` profile are included in the **Data Profile** drop-down.
4. Confirm that filter fields change when selecting profiles, and that the displayed fields are members of the selected profile.
5. Check the spacing of the grid layout in the **Filters** section. This will be rearranged in the future, as will spacing between sections, but the current items should flow properly.
